pyodide_build/buildpkg.py

```diff
@@ -171,7 +171,8 @@ def compile(path: Path, srcpath: Path, pkg: Dict[str, Any], args):
 
     post = pkg.get("build", {}).get("post")
     if post is not None:
-        site_packages_dir = srcpath / "install" / "lib" / "python3.8" / "site-packages"
+        pyfolder = ''.join(["python", os.environ.get("PYMAJOR","3"), ".", os.environ.get("PYMINOR","8")])
```

Comment:
Maybe we could add

```suggestion
        # use Python, 3.8 by default
        pyfolder = ''.join(["python", os.environ.get("PYMAJOR","3"), ".", os.environ.get("PYMINOR","8")])
```
so we can find it by grep the day we change the default python version.